app/pages/home/homePageController.ts

```diff
@@ -26,6 +34,61 @@ export class HomePageController {
     this.authService.withUser().then(() => {
       this.update();
     });
+
+    this.gettingStartedTourConfig = {
+      id: "getting-started-tour",
```

Comment:
I can understand the descriptions being in there. That has yet to be determined.